docs/source/reference/config.rst

```diff
@@ -131,7 +134,18 @@ Available fields and semantics:
     #   files to assign to these non-AWS instances).
     #
     # Default: 'LOCAL_CREDENTIALS'.
+    ### Format 1 ###
+    # A string; the same remote identity is applied to all launched resources.
     remote_identity: LOCAL_CREDENTIALS
+    ### Format 2 ###
+    # A dict mapping wildcard expression of cloud names to the resources to the
+    # resource identity.
+    # NOTE: If not a wildcard expression in the dict mapping does not match a
+    # cloud name for a resouce being deployed, the default remote identity is used.
+    # To specify your own default, utilize "*" as the wildcard expression.
+    remote_identity:
+      sky-serve-controller-*: my-controller-specific-identity
+      "*": SERVICE_ACCOUNT
```

Comment:
First attempt at how we represent multiple formats.

Reply:
Thanks for adding these! I reorganized the comments a little bit in the comment above, can we remove Format 1 and Format 2 here and use the comments above?